src/Components/LocationForm.js

```diff
@@ -1,22 +1,26 @@
 import { useState } from 'react';
 import styled from 'styled-components';
-import { useEffect } from 'react/cjs/react.development';
 
 import validateLocation from '../lib/Validation';
-/* import mapPin from '../lib/icons'; */
+
 import SelectIcons from './SelectIcons';
 export default function LocationForm({ onAddLocations }) {
   const initialLocation = {
     name: '',
     category: '',
     icon: '',
-    position: '',
-    adress: '',
+    latitude: '',
+    longitude: '',
+    address: '',
   };
 
   const [location, setLocation] = useState(initialLocation);
   const [isError, setIsError] = useState(false);
-  //console.log('added location', location);
+  /*  const [lat, setLat] = useState(null);
+  const [lng, setLng] = useState(null); */
```

Comment:
Same here, do not forget to delete this.

Reply:
Thank you, I'll delete it.

Reply:
Thanks a lot for your review!